modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -39,44 +58,26 @@ $ gadmin backup list
 
 In this example, the time range for backup `backup-2024-06-13T061939.131` is `[2024-06-13T06:19:39Z,2024-06-13T06:19:39Z]`, therefore, for a full backup is just a point. As for the incremental backup `backup-2024-06-13T063915.054`, its range is `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]`. `2024-06-13T06:35:00Z` is supported for it is covered by the time range `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]` of the backup `backup-2024-06-13T063915.054`. While, users can’t restore to the timepoint `2024-06-13T07:00:00Z` directly, for it is not in any backup's time range. However, if no restore was performed since `2024-06-13T07:00:00Z`, users may be able to create a new differential backup at right now and then restore to `2024-06-13T07:00:00Z`. 
 
+[NOTE]: Since the restore action will make any uncovered timepoints unavailable for restoration, it is recommended to ensure that all potential timepoints are covered by existing backups. If there is no backup available for the required timepoints, you can create one using the following command:
+[console, gsql]
+----
+gadmin backup create <backup_tag> --incremental --base <full_backup_tag>
+----
+This new backup will cover the time range from the creation time of the specified full backup <full_backup_tag> to the current time.
```

Comment:
@victorleeTG 

While it might seem obvious to some users, this section is intended to emphasize the importance of creating backups for those who may not be aware or might overlook this step. It helps ensure that users understand they can't recover data without having previously saved it, which is a critical point for anyone new to the process.